docs/api.yaml

```diff
@@ -127,8 +127,21 @@ paths:
           description: invalid user information or approval status
       security:
           - JWT: []
+  /recruitment:
+    get:
+      tags:
+        - recruitment
+      summary: Returns all recruitments available in application.
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      responses:
+        200:
+          description: All available recruitment data.
```

Comment:
Please add what recruitment data is being returned in response.

Reply:
oke

Reply:
![image](https://user-images.githubusercontent.com/34160783/114210214-cae6d380-995f-11eb-8f3e-a6bacd1eb264.png)

Reply:
Check swagger docs about how to properly describe responses: https://swagger.io/docs/specification/2-0/describing-responses/
In this case it should be something like that:
```
definitions:
    recruitmentData:
         type: object
         properties:
            id:
                type: integer
...
```

Reply:
@twrdyyy

Reply:
hold up

Reply:
omw